.github/codecov.yml

```diff
@@ -1,16 +1,17 @@
 comment:
   layout: "header, diff, components, files" # show component info in the PR comment
-component_management:
-  default_rules: # default rules that will be inherited by all components
-    statuses:
-      - type: project # in this case every component that doens't have a status defined will have a project type
-        informational: true
-        branches:
-          - master
-      - type: patch
+
+coverage:
+  status:
+    project:
+      default:
+        target: 45%
+        threshold: 5%
```

Comment:
This will model the following behavior - 

Target of 45% with a acceptable deviation of 5%. This will ensure that PRs which cause a minor drop below the 45% mark are not blocked from getting merged, while at the same time, dropping below 40% is a red flag which should be blocker and needs to be looked at. This is actually a pretty standard test threshold configuration style! Happy to talk more!